services/graphql-server/src/graphql/utils/embedded-image-tags.js

```diff
@@ -1,7 +1,13 @@
 const { extractEmbeddedTags } = require('@parameter1/base-cms-embedded-media');
 const { createAltFor, createSrcFor, createCaptionFor } = require('@parameter1/base-cms-image');
 
-module.exports = async (body, { imageHost, basedb }) => {
+const defaults = {
+  w: '1280',
+  fit: 'max',
+  auto: 'format,compress',
```

Comment:
Need to verify that the default compress of 45 doesn't make the image look like crap when not sending an enhanced format like avif or webp

Reply:
@zarathustra323 - I've tested a handful; they're all coming back as JPEGs and have looked identical.

Reply:
@brandonbk Tested with auto formatting disabled?